disable-luajit-file-download.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/cmake/Modules/GetLuaJIT.cmake b/cmake/Modules/GetLuaJIT.cmake
```

Comment:
On second thought, would there be any problem with putting the two patch files inside the `nix/` directory? That would make it more clear what they're used for and reduce clutter in the top-level directory.

Reply:
We really shouldn't have patch files for our own software stored inside our own repo.
We should just actually fix the problem or add the missing configuration mechanism the patch files are replacing.

Reply:
Oh, huh. I didn't even see this, but wound up stashing those in there, yeah. I do think we need to actually provide the configurations these patches are working around, but I don't think now is the time for it. Probably best done as a cleanup and rework of the whole pile of CMake scripts.

Reply:
I'll take this for now, but we can talk about how to do additional cleanups for the scripts as a next step.